frontend/app/components/layouts/protected-layout.tsx

```diff
@@ -49,6 +50,7 @@ function AppPageTitle() {
 function NavigationMenu() {
   const { t } = useTranslation(i18nNamespaces);
   const { SCCH_BASE_URI } = getClientEnv();
+  const origin = useUserOrigin();
```

Comment:
**Nitpick:** For consistency since we use `userOrigin` elsewhere.

```suggestion
  const userOrigin = useUserOrigin();
```

Reply:
fixed